x/auctions/types/genesis.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package types
+
+import (
+	"cosmossdk.io/errors"
+	"github.com/evmos/evmos/v19/utils"
+)
+
+// DefaultGenesisState sets default auctions genesis state.
+func DefaultGenesisState() *GenesisState {
+	return &GenesisState{
+		Params: Params{
+			EnableAuction: true,
+		},
+		Bid:   Bid{},
```

Comment:
this breaks the validation of the genesis
```suggestion
		Bid: Bid{
			Sender: "",
			Amount: sdk.NewCoin(utils.BaseDenom, sdk.ZeroInt()),
		},
```